Libraries/LibRegex/RegexByteCode.cpp

```diff
@@ -218,6 +218,18 @@ ALWAYS_INLINE ExecutionResult OpCode_FailForks::execute(MatchInput const& input,
     return ExecutionResult::Failed_ExecuteLowPrioForks;
 }
 
+ALWAYS_INLINE ExecutionResult OpCode_PopSaved::execute(MatchInput const& input, MatchState&) const
+{
+    if (!input.saved_positions.is_empty()) {
+        input.saved_positions.take_last();
+    }
+
+    if (!input.saved_code_unit_positions.is_empty()) {
+        input.saved_code_unit_positions.take_last();
+    }
+    return ExecutionResult::Failed_ExecuteLowPrioForks;
+}
```

Comment:
Let's exit early for these cases.
```suggestion
{
    if (input.saved_positions.is_empty() || input.saved_code_unit_positions.is_empty())
        return ExecutionResult::Failed_ExecuteLowPrioForks;

    input.saved_positions.take_last();
    input.saved_code_unit_positions.take_last();
    return ExecutionResult::Failed_ExecuteLowPrioForks;
}
```

The state is very broken if one is empty and the other isn't, so let's not further break it :)

Reply:
ok i think i fixed it.

i run locally  this ./Meta/ladybird.sh test
any other suggestion for the future for avoiding error like the missing empty check?

Reply:
You'd likely have to run wpt, but it's not a big deal - let the runner do that, we can always revert when needed or patch it up if an immediate fix is available.